aes-gcm/tests/common/mod.rs

```diff
@@ -4,7 +4,7 @@
 #[derive(Debug)]
 pub struct TestVector<K: 'static> {
     pub key: &'static K,
-    pub nonce: &'static [u8; 12],
+    pub nonce: &'static [u8],
```

Comment:
Is there a reason to make this non-fixed length ?

Reply:
Now we have nonces of two different lengths, so this is needed to be able to reuse the structure.